src/ucp/tag/rndv.c

```diff
@@ -420,7 +413,8 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_get_zcopy, (self),
         ucp_rndv_recv_data_init(rndv_req->send.rndv_get.rreq,
                                 rndv_req->send.length);
         ucp_rndv_req_send_rtr(rndv_req, rndv_req->send.rndv_get.rreq,
-                              rndv_req->send.rndv_get.remote_request);
+                              rndv_req->send.rndv_get.remote_request,
+                              rndv_req->send.length);
```

Comment:
before this PR it was rreq->recv.length, is that the same?

Reply:
no, this is different values
`rreq->recv.length` - posted by a receiver
`rndv_req->send.length` - received from RTS

here is just a fallback from GET to PUT scheme, so we should use `rndv_req->send.length` in order to use RNDV PUT instead of RNDV pipeline
we should post send

Reply:
what is the point to send back sender's buffer size?
how would the sender know the receive buffer size?

Reply:
ok, it's because when sender size is too big we just reject immediately? so it's to distinguish from pipelined protocol?

Reply:
> what is the point to send back sender's buffer size?
> how would the sender know the receive buffer size?

do you mean that we need to do `MIN(TAG_SEND::length; RTR::length)` on a sender side?
but then we send in an RTR packet a length that we couldn't really receive, since we register a receive buffer with the length = `RTS::length`

Reply:
> ok, it's because when sender size is too big we just reject immediately? 

yes, it's and we could use it directly w/o doing MIN

> so it's to distinguish from pipelined protocol?

yes, then a sender won't use RNDV pipeline protocol, since `TAG_SEND::length == RTR::length`
when a receiver is going to use RNDV pipeline protocol (e.g. a rceive buffer is CUDA buffer), it sends `RTR::length = MIN(frag_size, receive_buf_size)`